src/incidents/incidents-slice.ts

```diff
@@ -9,6 +9,11 @@ interface IncidentsState {
   status: 'loading' | 'completed'
 }
 
+export enum filter {
+  reported = 'reported',
+  all = 'all',
+}
+
```

Comment:
Let's move this to its own file. 

Also, we should declare this enum with a capital `F`.